tests/platform_helper/utils/test_versioning.py

```diff
@@ -117,6 +119,28 @@ def test_validate_template_version(template_check: Tuple[str, Type[BaseException
         assert (message % template_path) == str(exception.value)
 
 
+@pytest.mark.parametrize(
+    "template_check",
+    [
+        ("addon_different_version.yml", IncompatibleMajorVersion, ""),
+        ("addon_no_version.yml", ValidationException, "Template %s has no version information"),
+    ],
+)
+@patch("dbt_platform_helper.utils.versioning.get_file_app_versions")
+def test_validate_platform_helper_file_version(
+    get_file_app_versions, template_check: Tuple[str, Type[BaseException], str]
+):
+    get_file_app_versions.return_value = (1, 0, 0), (1, 0, 0)
+    template_name, raises, message = template_check
+
+    with pytest.raises(raises) as exception:
+        template_path = str(Path(f"{FIXTURES_DIR}/version_validation/{template_name}").resolve())
+        validate_platform_helper_file_version(template_path)
```

Comment:
I think setting `template_path` is part of the "arrange".
```suggestion
    template_name, raises, message = template_check
    template_path = str(Path(f"{FIXTURES_DIR}/version_validation/{template_name}").resolve())

    with pytest.raises(raises) as exception:
        validate_platform_helper_file_version(template_path)
```